packages/tables/resources/views/components/header-cell.blade.php

```diff
@@ -14,7 +14,7 @@
         @endif
         type="button"
         @class([
-            'flex items-center w-full px-4 py-2 whitespace-nowrap space-x-1 rtl:space-x-reverse font-medium text-sm text-gray-600',
+            'flex items-center w-full px-4 py-2 whitespace-nowrap font-medium text-sm text-gray-600',
```

Comment:
Wouldn't it be better/cleaner to use `gap-x-1` here instead? @danharrin @archilex @awcodes

Reply:
The point of the PR is that there are now more child elements of this button, so we can't use `space-` or `gap-`, we only need a left margin before the icon.

Reply:
We can use gap, since flex gap doesn't add an additional gap for the `sr-only` items, while `space-` does as it simply adds a hard margin to all child elements.

Reply:
Okay, make the change if you want but I don't think the margin is objectively bad, just a different approach

Reply:
Will create a PR.
To me `gap-x-1` is way more maintainable than `ml-1 rtl:ml-0 rtl:mr-1`.

Reply:
https://github.com/filamentphp/filament/pull/5725

Reply:
Agree @zepfietje and I learned something. Thanks.

Reply:
Perfect, thank you for fixing the issue initially!